.github/workflows/tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       max-parallel: 4
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Maybe we can enable testing on 3.9 as part of a separate PR @jaheba ?

Reply:
Also installing autogluon on python 3.9 appears to be giving issues in the test workflow @kashif so one more reason to split this up :-)